lib/ansible/modules/network/nxos/nxos_ntp.py

```diff
@@ -266,14 +275,21 @@ def config_ntp(delta, existing):
 
     ntp_cmds = []
     if peer_type:
+        if existing:
+            ntp_cmds.append('no ntp {0} {1}'.format(existing.get('peer_type'),
```

Comment:
I have been getting `no ntp None None` is one of the 9K devices. peer_type, address check is necessary here. Since you are using `get()` checking `existing` isn't enough as `get()` returns None if key doesn't exist.
`if existing.get('peer_type') and existing.get('address'):` instead of `if existing`.